doc/classes/AnimationNodeAdd3.xml

```diff
@@ -1,15 +1,22 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AnimationNodeAdd3" inherits="AnimationNode" category="Core" version="3.2">
 	<brief_description>
+		Blends two of three animations additively inside of an [AnimationNodeBlendTree].
 	</brief_description>
 	<description>
+		A resource to add to an [AnimationNodeBlendTree]. Blends two animations together additively out of three based on a value in the [-1.0, 1.0] range.
+		This node has three inputs:
+		- The base animation to add to
+		- A -add animation to blend with when the blend amount is in the [-1.0, 0.0] range.
+		- A +add animation to blend with when the blend amount is in the [0.0, 1.0] range
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="sync" type="bool" setter="set_use_sync" getter="is_using_sync">
+			If [code]true[/code], sets the [code]optimization[/code] to [code]false[/code] when calling [method AnimationNode.blend_input], forcing the blended animations update every frame.
```

Comment:
Should be "forcing the blended animations **to** update". There are several instances of this same description where "to" is missing.